core/modules/views/config/views.view.promoted.json

```diff
@@ -72,18 +73,33 @@
                             "operator": false
                         }
                     }
+                },
+                "empty": {
+                    "area": {
+                        "id": "area",
+                        "table": "views",
+                        "field": "area",
+                        "relationship": "none",
+                        "group_type": "group",
+                        "ui_name": "",
+                        "label": "",
+                        "empty": true,
+                        "content": "<p>No content has been promoted yet.</p>\n",
+                        "format": "filtered_html",
```

Comment:
It's unfortunate that we don't have any way of translating this on-the-fly currently. That's fine for now I think because we don't have config translation as of yet and this string is only shown once (and changeable by the user). So good enough for now I think.